src/nodes/display/PassNode.js

```diff
@@ -130,7 +131,18 @@ class PassNode extends TempNode {
 
 	}
 
-	setup() {
+	setup( { renderer } ) {
+
+		this.renderTarget.samples = this.options.samples === undefined ? renderer.samples : this.options.samples;
+
+		// Disable MSAA for WebGL backend for now
+		if ( renderer.backend.isWebGLBackend === true ) {
```

Comment:
Is this exception still required? My understand is you have added this bit since rendering multisampled to a color renderable texture does not yet work in the WebGL backend. But can at least the "normal" MSAA be used?

Reply:
Unfortunately yes it's still needed. I noticed that using PassNode with `samples > 0` doesn't seem to render anything in WebGL. I believe this might be due to the need for an additional frameBuffer somewhere in the render pipeline, which I couldn't easily resolve. That's why I added this comment.

Reply:
Could we add this in `WebGLBackend.beginRender()` for now?

Reply:
I wouldn't recommend so because basic `THREE.RenderTarget` setups works just fine `with samples > 0`. For example `webgpu_multisampled_renderbuffers.html`.

Reply:
Let's merge this PR then! It's possible do it in another PR, currently already a step forward in that sense. So the future goal is for us to move this and `isMultisampleRenderTargetTexture` into the renderer as @Mugen87 said, when this is ready examples like `webgpu_backdrop_water` should work too.